### ISSUE TITLE
ReferenceError: e is not defined

### ISSUE THREAD

**Issue by HC-Team** [Human-Connection/WebApp#153]
https://sentry.human-connection.org/hc/webapp-ym/issues/519/

```
ReferenceError: e is not defined
  at None (blob:https://alpha.human-connection.org/f0121df4-6c01-0a47-80df-884b0eda3eaa:1)
```

**Comment by visualjerk** [comment]
This is an issue with mapbox-gl and webpack: https://github.com/mapbox/mapbox-gl-js/issues/4359

I am testing the given solutions.

**Comment by visualjerk** [comment]
Fixed with #154
